cmd/fastly/main.go

```diff
@@ -39,11 +38,125 @@ func main() {
 		out            io.Writer = common.NewSyncWriter(os.Stdout)
 	)
 
+	// We have to manually handle the inclusion of the verbose flag here because
+	// Kingpin doesn't evaluate the provided arguments until app.Run which
+	// happens later in the file and yet we need to know if we should be printing
+	// output related to the application configuration file in this file.
+	var verboseOutput bool
+	for _, seg := range args {
+		if seg == "-v" || seg == "--verbose" {
+			verboseOutput = true
+		}
+	}
+
+	// Extract a subset of configuration options from the local application directory.
+	var file config.File
+	err := file.Read(configFilePath)
+	if err != nil {
+		if verboseOutput {
+			if err == config.ErrLegacyConfig {
+				text.Output(out, `
+					We found your local configuration file (required to use the CLI) was using a legacy format.
+					We will upgrade that file for you now.
```

Comment:
Just noting that I think this is the first time we've used a third person tone of voice "we" in the CLI output. And thus inconsistent with other messages. If thats our preferred tone going forward we should consider starting a document somewhere (internally or in this repo) with best practices for writing output messaging for the product. I personally found the use of "we" here strange 😉

Reply:
So something like this instead?

```
Found local configuration file...
File is being upgraded.
```